app/containers/HomePage/About.js

```diff
@@ -0,0 +1,357 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import * as img1 from 'images/Ico_01.svg';
+import * as img2 from 'images/Ico_02.svg';
+import * as img3 from 'images/Ico_03.svg';
+
+import * as img11 from 'images/1.png';
+import * as img22 from 'images/2.png';
+import * as img33 from 'images/3.png';
+import * as img44 from 'images/4.png';
+import * as img55 from 'images/5.png';
+
+import { SECOND_SCREEN, ANIMATE_IMAGE } from './constants';
+import messages from './messages';
+
+const Box = styled.section`
+  scroll-behavior: smooth;
+  text-align: center;
+  background-color: #fff;
+  padding-bottom: 80px;
+
+  .second-screen-header {
+    padding-top: 112px;
+    padding-bottom: 81px;
+    font-size: 30px;
+    font-family: OpenSansBold !important;
+  }
+
+  .separator {
+    transform: rotate(180deg);
+    height: 170px;
+  }
+
+  .main-difference {
+    li {
+      letter-spacing: -0.7px;
+      font-size: 18px;
+      color: #282828;
+
+      .card {
+        border: none;
+        .card-body {
+          padding: 0;
+        }
+      }
+
+      img {
+        height: 81px;
+      }
+
+      .item-title {
+        padding: 31px 0 26px 0;
+        font-family: OpenSansBold !important;
+      }
+
+      .item-content {
+        font-family: OpenSans;
+        padding: 0 3.25rem;
+      }
+    }
+  }
+
+  .second-screen-elements-column {
```

Comment:
class names should be more descriptive. Also styled components classes are scoped. you should be OK to give classes names that make sense within the scope of single component.